README.md

```diff
@@ -1,13 +1,17 @@
 # Navigate website
+
 Website developed using [React](https://reactjs.org/) + Typescript.
 
 This tool will be used to configure the telescope and its subsystems to point and track a celestial object during an observation. Navigate is a web application with a single centralized server that communicates with the different real-time control systems of the telescope. It also will communicate with other apps in the Gemini Program Platform (GPP) ecosystem, such as Observe and Chronicle.
 
 ## Launch on local development
+
 Make sure you have installed [NodeJS](https://nodejs.org/en/) in your machine.
 
 ### Using yarn
```

Comment:
The readme refers to using yarn or npm. But the lock-file is a pnpm-lock. So those are actually the options nobody should use! Maybe they can be replaced with just a single pnpm guide?

Reply:
Also pro tip, I use https://github.com/antfu/ni to just run `ni` and it will choose the correct package manager